### DISCUSSION TITLE
Is there a timeline for when dandisets with Zarr assets can be published?

### DISCUSSION THREAD

**Discussion by joshlawrimore** [dandi/helpdesk#165] [General]
I want to host derivative images of light sheet microscopy scans of intact mouse brain packaged as ome-zarr using the BIDS-microscopy standard. I noticed the [Publishing Dandisets](https://www.dandiarchive.org/handbook/14_publish/) section of the handbook states **"NOTE: Dandisets with Zarr assets currently cannot be published. We are actively working on enabling this feature."**

1. Does this mean data packaged in ome-zarr cannot be assigned a DOI at all?
2. Is there a timeline as to when datasets containing Zarr assets can be published?

**Comment by yarikoptic** [comment]
1. yes
2. no specific date. We are ATM still evaluating  https://earthmover.io/blog/icechunk and deciding on alternatively to proceed with custom implementation.

And alternative path, if the goal is just to get a DOI (not necessarily pointing to a specific version) could be  for us to prioritize https://github.com/dandi/dandi-archive/pull/2012 design/implementation and thus provide DOIs even for draft dandisets.

What is your ultimate goal behind getting a DOI -- immutability or just citeability?

**Reply by joshlawrimore** [reply]
Citeability. We use DOIs as a way of recording our work in the Data Science and Sharing Team at NIMH.

**Reply by yarikoptic** [reply]
Gotcha. We will prioritize finishing up that "DOI for drafts" functionality to start with.  Welcome to chime in a month or so if you do not hear back.

**Reply by joshlawrimore** [reply]
Awesome. Thank you so much!

**Reply by agt24** [reply]
Any news @yarikoptic ?

**Reply by yarikoptic** [reply]
I think we finished draft doi design doc. Might approach implementation next week or early next year.

**Reply by joshlawrimore** [reply]
Has there been any progress on getting DOIs for Dandisets containing ome-zarr assets?

**Reply by yarikoptic** [reply]
progress - yes. result - not yet.  Realistically, I think - within a month we should get "real DOIs" going for dandisets.  But one discovery is that unless they are non-draft, they would kinda be useless.   Adding a note: https://github.com/dandi/dandi-archive/pull/2012/files#r1926301678 .

**Reply by agt24** [reply]
Thanks @yarikoptic

Not 100% sure I follow the concerns about useless DOIs but I think DOIs on non-draft DandiSets are OK for our use case as long as they are immutable (+/- versioning ala Zenodo) and will be eventually citable.

**Reply by yarikoptic** [reply]
That's the catch, zarrs are mutable ATM. Dois will be citable though.

**Comment by agt24** [comment]
Hey @yarikoptic, 

Any news on making ZARR datasets DOI-able?
